taskana-adapter-camunda-listener/src/main/resources/sql/taskana-outbox-schema-oracle.sql

```diff
@@ -15,7 +15,10 @@ CREATE TABLE %schemaName%.EVENT_STORE
     ID NUMBER(32) NOT NULL,
     TYPE VARCHAR(32) NOT NULL,
     CREATED TIMESTAMP,
-    PAYLOAD VARCHAR2(4000)
+    PAYLOAD VARCHAR2(4000),
+    REMAINING_RETRIES NUMBER(32) NOT NULL,
+    BLOCKED_UNTIL TIMESTAMP,
```

Comment:
why is it NOT NULL in Postgres but not in Oracle?

Reply:
Changed